plugins/GSdx/GSdx.rc

```diff
@@ -111,39 +111,40 @@ IDB_PSX_NULL            BITMAP                  "res\\psx-logo-null.bmp"
 // Dialog
 //
 
-IDD_HACKS DIALOGEX 0, 0, 161, 235
+IDD_HACKS DIALOGEX 0, 0, 219, 225
 STYLE DS_SETFONT | DS_MODALFRAME | DS_FIXEDSYS | WS_POPUP | WS_CAPTION | WS_SYSMENU
 CAPTION "Hacks Configuration"
 FONT 8, "MS Shell Dlg", 400, 0, 0x1
 BEGIN
-    DEFPUSHBUTTON   "OK",IDOK,27,214,50,14
-    DEFPUSHBUTTON   "Cancel",IDCANCEL,84,214,50,14
```

Comment:
And change the X-offset of the OK and Cancel to:
```
    DEFPUSHBUTTON   "OK",IDOK,56,199,50,14
    DEFPUSHBUTTON   "Cancel",IDCANCEL,112,199,50,14
```

And then the PR can be squashed.